scripts/target/mame/mess.lua

```diff
@@ -3782,6 +3782,11 @@ files {
 	MAME_DIR .. "src/mame/video/comquest.cpp",
 }
 
+createMESSProjects(_target, _subtarget, "tectoy")
+files {
+	MAME_DIR .. "src/mame/drivers/pensebem.cpp",
+}
+
```

Comment:
I think it would make sense to put this in the VTech project.  Tectoy were basically an importer/distributor/localiser.  There’s a bit of a trade-off between having too many files in in a project, and too many project with one or two files.

Reply:
I very much disagree. TecToy have a notable list of releases in Brazil that were either developed solely by TecToy or developed in conjunction with other companies solely for release, distribution, and advertisement by TecToy. The Zeebo driver, for instance, would be right at home in a tectoy-specific subtarget.

Reply:
The original Smart Start was developed by VTech. And the 80s original release of Pense Bem by TecToy was merely a rebranding to be distributed in the Brazilian market. But the 2017 re-release of it is running on a new hardware architecture and it seems to have been designed/developed by TecToy, not VTech.

Reply:
I've just added Zeebo as well as titan_soc (SoC used in TecToy's Mega Drive 4) to this tectoy-specific subtarget.

Reply:
ooops! Now I see that the PR had already been merged 20 minutes ago, so adding Zeebo and titan_soc to the subtarget didn't make into the merge. We may do it separately in a subsequent PR then...